### PR TITLE
updated version compatibility in readme.md

### PR DESCRIPTION
**Pull request by Chrismelba-Instaclustr**

### CODE REVIEW

**Review by peter-toms — DISMISSED** [review]

**Comment by peter-toms** [comment]
adding more people as reviewers to this PR

**Review by Quintero-daniel — COMMENTED** [review]
[Oportunistic] Can you please fix the typo (Instacluster) in line 96

File: Makefile [hunk 1]
```diff
@@ -1,7 +1,7 @@
 
 BIN_NAME="terraform-provider-instaclustr"
 
-VERSION=1.9.7
+VERSION=1.9.10
```
Comment (hail-insta) [inline Makefile:4]: why the big bump?

**Comment by hail-insta** [comment]
There's a conflict that needs to be fixed & should pull in master to reflect the recent workflow change

**Review by hail-insta — APPROVED** [review]

**Review by peter-toms — APPROVED** [review]

**Pull request merged**